validator/engine/validator_test.js

```diff
@@ -85,9 +86,12 @@ function findHtmlFilesRelativeToTestdata() {
   for (const root of process.env['TESTDATA_ROOTS'].split(':')) {
     if (path.basename(root) === 'extensions') {
       for (const extension of readdir(root)) {
-        const testPath = path.join(extension, '0.1', 'test');
-        if (isdir(path.join(root, testPath))) {
-          testSubdirs.push({root: root, subdir: testPath});
+        const extensionVersions = ['0.1', '1.0'];
```

Comment:
Actually I'm going to change this back to being explicit about version numbers rather than just every directory within the extension name.